winui/Cartesian-Charts/Zooming-and-panning.md

```diff
@@ -291,39 +216,6 @@ chart.Behaviors.Add(zooming);
 
 ![Zoom mode support in WinUI Chart](Zooming-and-panning_images/WinUI_chart_zoom_modeY.png)
 
-## Maximum Zoom level
-
-The [MaximumZoomLevel](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartZoomPanBehavior.html#Syncfusion_UI_Xaml_Charts_ChartZoomPanBehavior_MaximumZoomLevel) property defines the maximum zooming level of the chart area. Zooming will be stopped after reaching this value.
-
-{% tabs %}
-
-{% highlight xaml %}
-
-<chart:SfCartesianChart>
-    <chart:SfCartesianChart.Behaviors>
-        <chart:ChartZoomPanBehavior EnableSelectionZooming="True" MaximumZoomLevel="100"/>
-    </chart:SfCartesianChart.Behaviors>
-    ...
-</chart:SfCartesianChart>
-{% endhighlight %}
-
-{% highlight c# %}
-
-SfCartesianChart chart = new SfCartesianChart();
-...
-ChartZoomPanBehavior zooming = new ChartZoomPanBehavior()
-{
-    EnableSelectionZooming = true,
-    MaximumZoomLevel = 100
-};
-
-chart.Behaviors.Add(zooming);
-...
-
-{% endhighlight %}
-
-{% endtabs %}
-
 ## Enable Panning
 
 Panning feature allows moving the visible area of the chart when it is zoomed in. To enable panning, you have to set [EnablePanning](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartZoomPanBehavior.html#Syncfusion_UI_Xaml_Charts_ChartZoomPanBehavior_EnablePanning) property to `true`.
```

Comment:
Panning feature allows moving the visible area of the chart when it is zoomed in. To enable panning, you have to set the [EnablePanning](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartZoomPanBehavior.html#Syncfusion_UI_Xaml_Charts_ChartZoomPanBehavior_EnablePanning) property to `true`.